src/ui/osx/TogglDesktop/MainWindowController.m

```diff
@@ -476,4 +474,22 @@ -(void) handleContinueSignInNotification:(NSNotification *) noti
         [self.loginViewController continueSignIn];
     }
 }
+
+-(void) displayLoginViewController:(BOOL) display
+{
+    if (display)
```

Comment:
I didn't get why is this a single method, these are two separate branches of code and it seems it'd be easier to keep them as 2 separate methods.